src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -656,6 +658,44 @@ internal override bool IsCallerMemberName
             }
         }
 
+        private int? _cachedCallerArgumentExpressionParameterIndex;
```

Comment:
>private int? _cachedCallerArgumentExpressionParameterIndex; [](start = 8, length = 59)

Please place the field declaration at the beginning of the class declaration, next to other fields. Also, we use "lazy" prefix for fields with delayed initialization.